flex/__init__.py

```diff
@@ -1,3 +1,9 @@
 VERSION = '6.3.0'
 
-from flex.core import load  # NOQA
+
+def load(*args, **kwargs):
```

Comment:
This is done so that it's still backward compatible, but allowing us to have one version in all files